test/Electrical/dc_motor.jl

```diff
@@ -0,0 +1,219 @@
+using ModelingToolkitStandardLibrary.Electrical
+using ModelingToolkitStandardLibrary.Mechanical.Rotational
+using ModelingToolkitStandardLibrary.Blocks
+import ModelingToolkitStandardLibrary
+using ModelingToolkit, OrdinaryDiffEq, Test
+# using Plots
+
+@parameters t
+
+@testset "DC motor" begin
+    R = 0.5
+    L = 4.5e-3
+    k = 0.5
+    J = 0.02
+    f = 0.01
+    V_step = 10
+    tau_L_step = -3
+    @named ground = Ground()
+    @named source = SignalVoltage()
+    @named voltage_step = Blocks.Step(height=V_step, start_time=0)
+    @named R1 = Resistor(R=R)
+    @named L1 = Inductor(L=L)
+    @named emf = EMF(k=k)
+    @named fixed = Fixed()
+    @named load = Torque(use_support=false)
+    @named load_step = Blocks.Step(height=tau_L_step, start_time=3)
+    @named inertia = Inertia(J=J)
+    @named friction = ViscousFriction(f=f)
+
+    connections = [
+        connect(fixed.flange, emf.support, friction.flange_b)
+        connect(emf.flange, friction.flange_a, inertia.flange_a)
+        connect(inertia.flange_b, load.flange)
+        connect(load_step.output, load.tau)
+        connect(voltage_step.output, source.input)
+        connect(source.p, R1.p)
+        connect(R1.n, L1.p)
+        connect(L1.n, emf.p)
+        connect(emf.n, source.n, ground.g)
+    ]
+
+    @named model = ODESystem(connections, t, systems=[ground, voltage_step, source, R1, L1, emf, fixed, load, load_step, inertia, friction])
+    sys = structural_simplify(model)
```

Comment:
@YingboMa It seems that the last addition to MTK broke this example. Which already worked. 

With MTKv8.13.1 I get these, which look somewhat incorrect:
```julia
julia> equations(sys)
6-element Vector{Equation}:
 Differential(t)(L1₊i(t)) ~ L1₊v(t) / L1₊L
 Differential(t)(inertia₊phi(t)) ~ inertia₊w(t)
 Differential(t)(inertia₊w(t)) ~ inertia₊a(t)
 Differential(t)(inertia₊phi(t)) ~ -friction₊phi_relˍt(t)
 Differential(t)(inertia₊w(t)) ~ inertia₊phiˍtt(t)
 Differential(t)(inertia₊phi(t)) ~ emf₊phiˍt(t)
```

With MTKv8.11.6 I get this, and the solutions looks correct too (see plot I posted).
```julia
julia> equations(sys)
6-element Vector{Equation}:
 Differential(t)(L1₊i(t)) ~ L1₊v(t) / L1₊L
 Differential(t)(emf₊phi(t)) ~ emf₊w(t)
 Differential(t)(inertia₊phi(t)) ~ inertia₊w(t)
 Differential(t)(inertia₊w(t)) ~ inertia₊a(t)
 Differential(t)(friction₊phi_rel(t)) ~ friction₊w_rel(t)
 Differential(t)(friction₊w_rel(t)) ~ friction₊a_rel(t)
```

Reply:
Why do they look incorrect?

Reply:
Here is the model, maybe it can serve as an example:
```julia
begin
    var"##iv#638" = (@variables(t))[1]
    var"##sts#639" = (collect)(
        @variables(
            ground₊g₊v(t),
            ground₊g₊i(t),
            voltage_step₊output₊u(t),
            source₊v(t),
            source₊i(t),
            source₊p₊v(t),
            source₊p₊i(t),
            source₊n₊v(t),
            source₊n₊i(t),
            source₊input₊u(t),
            R1₊v(t),
            R1₊i(t),
            R1₊p₊v(t),
            R1₊p₊i(t),
            R1₊n₊v(t),
            R1₊n₊i(t),
            L1₊v(t),
            L1₊i(t),
            L1₊p₊v(t),
            L1₊p₊i(t),
            L1₊n₊v(t),
            L1₊n₊i(t),
            emf₊v(t),
            emf₊i(t),
            emf₊phi(t),
            emf₊w(t),
            emf₊p₊v(t),
            emf₊p₊i(t),
            emf₊n₊v(t),
            emf₊n₊i(t),
            emf₊flange₊phi(t),
            emf₊flange₊tau(t),
            emf₊support₊phi(t),
            emf₊support₊tau(t),
            fixed₊flange₊phi(t),
            fixed₊flange₊tau(t),
            load₊phi_support(t),
            load₊flange₊phi(t),
            load₊flange₊tau(t),
            load₊tau₊u(t),
            load_step₊output₊u(t),
            inertia₊phi(t),
            inertia₊w(t),
            inertia₊a(t),
            inertia₊flange_a₊phi(t),
            inertia₊flange_a₊tau(t),
            inertia₊flange_b₊phi(t),
            inertia₊flange_b₊tau(t),
            friction₊phi_rel(t),
            friction₊w_rel(t),
            friction₊a_rel(t),
            friction₊tau(t),
            friction₊flange_a₊phi(t),
            friction₊flange_a₊tau(t),
            friction₊flange_b₊phi(t),
            friction₊flange_b₊tau(t)
        )
    )
    var"##ps#640" = (collect)(
        @parameters(
            voltage_step₊offset,
            voltage_step₊start_time,
            voltage_step₊height,
            R1₊R,
            L1₊L,
            emf₊k,
            load_step₊offset,
            load_step₊start_time,
            load_step₊height,
            inertia₊J,
            friction₊f
        )
    )
    var"##eqs#641" = [
        (Symbolics.connect)(fixed.flange, emf.support, friction.flange_b)
        (Symbolics.connect)(emf.flange, friction.flange_a, inertia.flange_a)
        (Symbolics.connect)(inertia.flange_b, load.flange)
        (Symbolics.connect)(load_step.output, load.tau)
        (Symbolics.connect)(voltage_step.output, source.input)
        (Symbolics.connect)(source.p, R1.p)
        (Symbolics.connect)(R1.n, L1.p)
        (Symbolics.connect)(L1.n, emf.p)
        (Symbolics.connect)(emf.n, source.n, ground.g)
        ground₊g₊v ~ 0
        voltage_step₊output₊u ~
            voltage_step₊offset +
            (IfElse.ifelse)(t < voltage_step₊start_time, 0, voltage_step₊height)
        source₊v ~ -1source₊n₊v + source₊p₊v
        0 ~ source₊n₊i + source₊p₊i
        source₊i ~ source₊p₊i
        source₊v ~ source₊input₊u
        R1₊v ~ -1R1₊n₊v + R1₊p₊v
        0 ~ R1₊n₊i + R1₊p₊i
        R1₊i ~ R1₊p₊i
        R1₊v ~ R1₊R * R1₊i
        L1₊v ~ -1L1₊n₊v + L1₊p₊v
        0 ~ L1₊n₊i + L1₊p₊i
        L1₊i ~ L1₊p₊i
        (Differential(t))(L1₊i) ~ L1₊v / L1₊L
        emf₊v ~ -1 * emf₊n₊v + emf₊p₊v
        0 ~ emf₊n₊i + emf₊p₊i
        emf₊i ~ emf₊p₊i
        emf₊phi ~ -1 * emf₊support₊phi + emf₊flange₊phi
        (Differential(t))(emf₊phi) ~ emf₊w
        emf₊k * emf₊w ~ emf₊v
        emf₊flange₊tau ~ -1 * emf₊k * emf₊i
        fixed₊flange₊phi ~ 0.0
        load₊phi_support ~ 0
        load₊flange₊tau ~ -1load₊tau₊u
        load_step₊output₊u ~
            load_step₊offset +
            (IfElse.ifelse)(t < load_step₊start_time, 0, load_step₊height)
        inertia₊phi ~ inertia₊flange_a₊phi
        inertia₊phi ~ inertia₊flange_b₊phi
        (Differential(t))(inertia₊phi) ~ inertia₊w
        (Differential(t))(inertia₊w) ~ inertia₊a
        inertia₊J * inertia₊a ~ inertia₊flange_a₊tau + inertia₊flange_b₊tau
        friction₊phi_rel ~ -1 * friction₊flange_a₊phi + friction₊flange_b₊phi
        (Differential(t))(friction₊phi_rel) ~ friction₊w_rel
        (Differential(t))(friction₊w_rel) ~ friction₊a_rel
        friction₊flange_b₊tau ~ friction₊tau
        friction₊flange_a₊tau ~ -1 * friction₊tau
        friction₊tau ~ friction₊f * friction₊w_rel
    ]
    var"##defs#642" = (Dict)(
        (Pair)(R1₊i, 0.0),
        (Pair)(load_step₊start_time, 3),
        (Pair)(inertia₊flange_b₊phi, 0.0),
        (Pair)(friction₊flange_b₊tau, 0.0),
        (Pair)(friction₊a_rel, 0.0),
        (Pair)(friction₊phi_rel, 0.0),
        (Pair)(load₊tau₊u, 0.0),
        (Pair)(inertia₊flange_a₊tau, 0.0),
        (Pair)(emf₊p₊v, 1.0),
        (Pair)(L1₊n₊v, 1.0),
        (Pair)(load₊flange₊tau, 0.0),
        (Pair)(friction₊tau, 0.0),
        (Pair)(load_step₊output₊u, 0.0),
        (Pair)(voltage_step₊output₊u, 0.0),
        (Pair)(friction₊flange_b₊phi, 0.0),
        (Pair)(L1₊p₊i, 1.0),
        (Pair)(source₊v, 0.0),
        (Pair)(emf₊n₊i, 1.0),
        (Pair)(L1₊n₊i, 1.0),
        (Pair)(friction₊w_rel, 0.0),
        (Pair)(L1₊L, 0.0045),
        (Pair)(inertia₊flange_a₊phi, 0.0),
        (Pair)(inertia₊J, 0.02),
        (Pair)(friction₊f, 0.01),
        (Pair)(load_step₊height, -3),
        (Pair)(emf₊v, 0.0),
        (Pair)(emf₊n₊v, 1.0),
        (Pair)(L1₊p₊v, 1.0),
        (Pair)(emf₊i, 0.0),
        (Pair)(emf₊support₊tau, 0.0),
        (Pair)(R1₊n₊i, 1.0),
        (Pair)(emf₊phi, 0.0),
        (Pair)(load_step₊offset, 0),
        (Pair)(R1₊v, 0.0),
        (Pair)(ground₊g₊i, 1.0),
        (Pair)(L1₊i, 0.0),
        (Pair)(inertia₊flange_b₊tau, 0.0),
        (Pair)(emf₊p₊i, 1.0),
        (Pair)(source₊i, 0.0),
        (Pair)(emf₊support₊phi, 0.0),
        (Pair)(R1₊p₊i, 1.0),
        (Pair)(emf₊k, 0.5),
        (Pair)(R1₊R, 0.5),
        (Pair)(load₊flange₊phi, 0.0),
        (Pair)(emf₊w, 0.0),
        (Pair)(inertia₊phi, 0.0),
        (Pair)(source₊p₊v, 1.0),
        (Pair)(L1₊v, 0.0),
        (Pair)(voltage_step₊start_time, 0),
        (Pair)(source₊n₊i, 1.0),
        (Pair)(emf₊flange₊phi, 0.0),
        (Pair)(R1₊n₊v, 1.0),
        (Pair)(voltage_step₊offset, 0),
        (Pair)(ground₊g₊v, 1.0),
        (Pair)(voltage_step₊height, 10),
        (Pair)(source₊p₊i, 1.0),
        (Pair)(fixed₊flange₊tau, 0.0),
        (Pair)(fixed₊flange₊phi, 0.0),
        (Pair)(friction₊flange_a₊tau, 0.0),
        (Pair)(friction₊flange_a₊phi, 0.0),
        (Pair)(R1₊p₊v, 1.0),
        (Pair)(source₊input₊u, 0.0),
        (Pair)(load₊phi_support, 0.0),
        (Pair)(inertia₊w, 0.0),
        (Pair)(source₊n₊v, 1.0),
        (Pair)(emf₊flange₊tau, 0.0),
        (Pair)(inertia₊a, 0.0),
    )
    var"##iv#643" = (@variables(t))[1]
    (ODESystem)(
        var"##eqs#641",
        var"##iv#643",
        var"##sts#639",
        var"##ps#640";
        defaults = var"##defs#642",
        name = :model,
        checks = false,
    )
end
```